stage #2/module #4 String/lesson #1. String/content.md

```diff
@@ -0,0 +1,200 @@
+| String(byte[] bytes, int offset, int length)                     | Constructs a new String by decoding the specified subarray of bytes using the platform's default charset.                                                                                   |
+| String(byte[] bytes, int offset, int length, Charset charset)    | Constructs a new String by decoding the specified subarray of bytes using the specified charset.                                                                                            |
+| String(byte[] bytes, int offset, int length, String charsetName) | Constructs a new String by decoding the specified subarray of bytes using the specified charset.                                                                                            |
+| String(byte[] bytes, String charsetName)                         | Constructs a new String by decoding the specified array of bytes using the specified charset.                                                                                               |
+| String(char[] value)                                             | Allocates a new String so that it represents the sequence of characters currently contained in the character array argument.                                                                |
+| String(char[] value, int offset, int count)                      | Allocates a new String that contains characters from a subarray of the character array argument.                                                                                            |
+| String(int[] codePoints, int offset, int count)                  | Allocates a new String that contains characters from a subarray of the Unicode code point array argument.                                                                                   |
+| String(String original)                                          | Initializes a newly created String object so that it represents the same sequence of characters as the argument; in other words, the newly created string is a copy of the argument string. |
+| String(StringBuffer buffer)                                      | Allocates a new string that contains the sequence of characters currently contained in the string buffer argument.                                                                          |
+| String(StringBuilder builder)                                    | Allocates a new string that contains the sequence of characters currently contained in the string builder argument.                                                                         |
+
+## String pool
+
+### Overview
+
+The **String** is the most widely used data structure. Caching the String literals and reusing them
+saves a lot of heap space because different String variables refer to the same object in the String
+pool. String intern pool serves exactly this purpose.
```

Comment:
The paragraph should have continuous text without breaks, so that at different screen sizes the text wraps on the right words.
If I'm not wrong, you broke paragraph on 3 lines. In this case it will be displayed in 3+ lines started with words "The", "saves" and "pool"

Reply:
![image](https://user-images.githubusercontent.com/28488042/159910441-12e4e140-aa35-43e5-b486-7ae2aedb353f.png)
You are wrong, I broke the paragraph into 3 lines, but it is displayed on 2 at my screen resolution.